From e772cfd03d23e10a3db00857b6c28ffe0b4f40cd Mon Sep 17 00:00:00 2001
From: Ljzkirito <kirito.ljz@gmail.com>
Date: Tue, 20 Apr 2021 11:37:02 +0800
Subject: [PATCH] luci use theme-argon instead of theme-bootstrap

---
 collections/luci/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/collections/luci/Makefile b/collections/luci/Makefile
index 868d72113a3..efdd4b930b7 100644
--- a/collections/luci/Makefile
+++ b/collections/luci/Makefile
@@ -12,7 +12,7 @@ LUCI_BASENAME:=luci
 LUCI_TITLE:=LuCI interface with Uhttpd as Webserver (default)
 LUCI_DESCRIPTION:=Standard OpenWrt set including full admin with ppp support and the default Bootstrap theme
 LUCI_DEPENDS:= \
-	+uhttpd +uhttpd-mod-ubus +luci-mod-admin-full +luci-theme-bootstrap \
+	+uhttpd +uhttpd-mod-ubus +luci-mod-admin-full +luci-theme-argon \
 	+luci-app-firewall +luci-app-opkg +luci-proto-ppp +libiwinfo-lua +IPV6:luci-proto-ipv6 \
 	+rpcd-mod-rrdns
 